Spanner/src/SpannerClient.php

```diff
@@ -566,6 +578,14 @@ function (array $instance) {
      * $database = $spanner->connect('instance-id', 'database-id');
      * ```
      *
+     * Database role configured on the $options parameter
+     * will be applied to the session created by this object.
+     * Note: When the databseRole and sessionPool both are present in the options,
+     * we prioritize the sessionPool.
+     * ```
+     * $database = $spanner->connect('instance-id', 'database-id', ['databaseRole' => 'Reader']);
+     * ```
+     *
```

Comment:
I can't comment on the line below, but we should also add databaseRole to the documented configuration options.

Reply:
Yes, will add that in the configuration options in the comment